app/controllers/healthcheck_controller.rb

```diff
@@ -5,4 +5,13 @@ def overdue
     logger.error "HealthcheckController#overdue: #{e.message}"
     render json: { overdue: nil }, status: :service_unavailable
   end
+
+  def unenqueued_scheduled_editions
+    render json: {
+      unenqueued_scheduled_editions: Healthcheck::ScheduledPublishing.new.unenqueued_editions,
+    }
+  rescue ActiveRecord::StatementInvalid => e
```

Comment:
Since both controller methods catch this, is there a way we can do a [`rescue_from`](https://apidock.com/rails/ActiveSupport/Rescuable/ClassMethods/rescue_from) to reduce the duplication?

Reply:
I chose not to `rescue_from` here as the JSON is slightly different in the two methods.  Do you know if there's a way round this?

Reply:
Hmm that's true, I wonder if the `overdue` or `unenqueued_scheduled_editions` field is superfluous since it's already in the path and we could just have a `value` field. However, this feels a bit out of scope for this given that it requires changing the other one too, so we can leave it like this for now.